protocol-models/bin/generate-typescript-classes.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+git config --global --add safe.directory /airbyte-protocol
+
+ROOT_DIR=${ROOT_DIR:-$(git rev-parse --show-toplevel)}
+cd $ROOT_DIR
+
+source ".env"
+echo "loaded VERSION=$VERSION"
+
+[ -z "$ROOT_DIR" ] && exit 1
+
+cd protocol-models/typescript
+
+# package.json's vesion MUST be in the file as a literal... so lets use jq to replace it.
+jq ".version=\"$VERSION\"" package.json > package.json.tmp
+mv package.json.tmp package.json
```

Comment:
This is rather jank.... any suggestions?

Reply:
I've done a similarly jank thing before
I guess a "feature" of this jank is it's more likely to work across different Unices 😄 
https://github.com/airbytehq/airbyte-platform-internal/blob/edf68504ebe430d349d3815bc341fe35902f37c7/tools/bin/update_oss_reference#L139-L143

If you want something that looks fancier you can check out this link https://unix.stackexchange.com/a/186126 but I feel like it's fine as is

Reply:
You can just use `(p)npm version <new-version>` to set the version in the package.json.